### ISSUE TITLE
A method for filtering objectionable content

### ISSUE THREAD

**Issue by codesworth** [entaq/floq#39]
We would have to add this too. I think it will be appropriate to add it in settings to allow for users to either see content with sensitive media or not.

Also before a cliq is added we may have to add a field that indicates that the cliq may contain sensitive media

So we can filter public cliqs by user sensitive media Preference

**Comment by jberkey78** [comment]
That’s difficult to manage, especially when some people have different standards when it comes to sensitive content. It’s a good idea though and when private cliqs are available that could be a feature. Another option is to tag a photo as potentially sensitive at time of uploading and blur the image with a hide icon which while require someone to tap to see or have a setting that allows them to see. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by codesworth** [comment]
Yeah. Lets see. but We would have to implement this before first release
